todos/views.py

```diff
@@ -32,4 +36,16 @@ def update(request, todo_id):
     todo.isCompleted = isCompleted
 
     todo.save()
-    return redirect('todos:index')
\ No newline at end of file
+    return redirect('todos:index')
+
+class RegisterView(generic.CreateView):
+    form_class = UserCreationForm
```

Comment:
Where is the minimum check length of the password. We want the password length to be of 6 characters atleast